net/vpn-policy-routing/Makefile

```diff
@@ -4,8 +4,8 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=vpn-policy-routing
-PKG_VERSION:=0.3.2
-PKG_RELEASE:=20
+PKG_VERSION:=0.3.4
+PKG_RELEASE:=8
```

Comment:
```suggestion
PKG_RELEASE:=1
```

Reply:
Thanks for the comment, because the binaries are also available from my package repo and not to downgrade the version, it can either be 0.3.4-8 or I can bring it further up to 0.3.5.-1 if it makes any difference.